andela/travel_tool_front
# #161975169 Fix badge on the approvals modal upon approval

#### What does this PR do?
Fixes issue badge on the approvals modal that does not change for approved requests

#### Description of Task to be completed?
- An approved request should have the ```Travel Stage``` badge in the approvals modal.

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- checkout the branch `git checkout ch-change-badge-for-approved-requests-161975169`
- Start the server with `make start`
- Start the backend of this project using this link [Click here](https://github.com/andela/travel_tool_back)
- Visit https://travela-local.andela.com
- Login to the application
- Navigate to the request page
- Create a new request.
- The manager should approve the request.
- View the approval, the badge should show ```Travel Stage```

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-11-16 at 12 01 02" src="https://user-images.githubusercontent.com/12511966/48611469-63e3b100-e997-11e8-92c8-ff0a27add4f7.png">

#### Questions:
